config.default.yaml

```diff
@@ -334,6 +334,14 @@ renewable:
     extendable: true
     csp_model: advanced # simple or advanced
 
+
+# clean_cooking configuration
+clean_cooking:
+  enable: true
+  fuel: ["AC", "lpg", "pellets", "firewood", "charcoal", "heat"]
+  p_nom: [1, 1, 1, 1, 1, 1] # same order as fuels
```

Comment:
Do you need to specify p_nom and p_max_pu here?

Reply:
Suggestion. Try out some minimal examples to understand what p_nom and p_nom_pu mean